tests/integration/heartbeat/heartbeat.spec.ts

```diff
@@ -44,22 +44,24 @@ describe('heartbeat', function () {
 
   describe('Happy Path', function () {
     it('pulse should return 200 status code and save the heartbeat pulse', async function () {
-      const id = '1';
+      const id = '513a40b2-783f-4c82-bd1a-5b6a7aca75d5';
```

Comment:
Use [faker](https://www.npmjs.com/package/@faker-js/faker) package in order to generate uuid instead of putting hard-coded values